### PR TITLE
Feat/civ 449

### PR DESCRIPTION
**Pull request by zikrurR**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-449

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -145,7 +145,7 @@ doc_assembly:
   oconFormOrganisationWithoutDQs: "CV-CMC-GOR-ENG-0015.docx"
   noticeOfTransferToCcbcSentToDefendantTemplateId: "CV-CMC-GOR-ENG-0016.docx"
   paperResponseAdmissionTemplateId: "CV-CMC-GOR-ENG-0016.docx"
-  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630.docx"
+  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630-1.odt"
```
Comment (mounikahmcts) [inline src/main/resources/application.yml:148]: Why the doc name is .odt

**Review by mounikahmcts — DISMISSED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (zikrurR) [reply src/main/resources/application.yml]: As discussed, this will be handled via [CIV-7013](https://tools.hmcts.net/jira/browse/CIV-7013)

**Review by asthamalviya — APPROVED** [review]

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -145,7 +145,7 @@ doc_assembly:
   oconFormOrganisationWithoutDQs: "CV-CMC-GOR-ENG-0015.docx"
   noticeOfTransferToCcbcSentToDefendantTemplateId: "CV-CMC-GOR-ENG-0016.docx"
   paperResponseAdmissionTemplateId: "CV-CMC-GOR-ENG-0016.docx"
-  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630.docx"
+  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630-1.odt" # TODO change to docx format
```
Comment (sabahirfan) [inline src/main/resources/application.yml:148]: this TODO seems to be pending.

Comment (zikrurR) [reply src/main/resources/application.yml:148]: This was added by me. There was a water mark when creating a docx document from my machine.

**Review by sabahirfan — APPROVED** [review]

**Pull request merged**